workspace.spec

```diff
@@ -572,6 +572,378 @@ module Workspace {
 		string description;
 	} ProvenanceAction;
 
+	/* Credit Metadata-related objects and fields
+
+		Workspace objects have a list of zero or more sets of credit metadata (CM), or citation information for the object. This information allows data imported into KBase to retain the appropriate details for users wishing to cite the data source, and for those who contributed to its creation to be credited for their work.
```

Comment:
Please line wrap @ 100 chars

Reply:
Once text changes are finalised I will implement line wrap throughout.

Reply:
This has been done, I'll recheck on the final review